src/tpm/kmyth_seal_unseal_impl.c

```diff
@@ -28,21 +27,90 @@
  */
 extern const cipher_t cipher_list[];
 
+//############################################################################
```

Comment:
This function seems like more of a "formatting" or "TPM interface" utility. My preference would be to move it, say, to utils/src/formatting_tools.c.

Reply:
This was done.